modules/journal/view/ajax.php

```diff
@@ -0,0 +1,25 @@
+ <?
+includelogic("journal/search");
+includelogic('accounting/accounting');
```

Comment:
do not remove, add a comment that we need it for use in search_openpost_accountplan() where it is referenced as a global